src/main/java/org/usfirst/frc4904/robot/subsystems/Hood.java

```diff
@@ -0,0 +1,118 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.Util;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CustomPIDController;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
+import org.usfirst.frc4904.standard.custom.sensors.CANEncoder;
+import org.usfirst.frc4904.standard.custom.sensors.CustomDigitalLimitSwitch;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.subsystems.motor.PositionSensorMotor;
+
+public class Hood extends PositionSensorMotor {
+    public static enum HoodStatus {
+        IDLE, MOVING, IN_POSITION
+    }
+
+    public final double DEFAULT_SPEED = 0;
+    private final double LOWER_HOOD_ANGLE = 0; //TODO: Add this value
+    private final double RANGE_HOOD_ANGLES = 35.0;
+    private final double UPPER_HOOD_ANGLE = LOWER_HOOD_ANGLE + RANGE_HOOD_ANGLES; //TODO: Does having all of these theoretical constants negate the zeroing we're doing?
```

Comment:
these constants do not negate zeroing. The zeroing is because although the encoder on the servo is absolute, "0" will be a different place every time

Reply:
I know, I more was wondering if perhaps the angle difference between the lower and upper part of the hood could change over time (if teeth get damaged or something else). For now, we should leave it as is, however, I'd be curious to see what the actual hood rotation amount is.